sdk/ai/azure-ai-generative/setup.py

```diff
@@ -42,7 +42,7 @@
     url="https://github.com/Azure/azure-sdk-for-python",
     keywords="azure, azuresdk, azure sdk",
     classifiers=[
-        "Development Status :: 4 - Beta",
+        "Development Status :: 7 - Inactive",
```

Comment:
Was this intentional? It probably should be a different PR....

Reply:
Oh, good call, I did not notice it.